### PR TITLE
ACC-1169 Default branch name change chores

### PR DESCRIPTION
**Pull request by magsallen**
## Description
Chore: Following the branch name change, this PR handles associated chores as per https://github.com/Financial-Times/next/wiki/Migrating-apps-to-use-main-branch-(instead-of-master)

Includes:
- Updating the default branch name in CircleCI config
- Updates other references to the default branch name
- Updates n-gage from v3 to v8
- Updates node from v8 to v12
- Updates husky scripts in package.json

## Ticket
https://financialtimes.atlassian.net/browse/ACC-1169

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
     working_directory: ~/project/build
     docker:
-      - image: circleci/node:8-browsers
+      - image: circleci/node:12-browsers
 
-  container_config_lambda_node8: &container_config_lambda_node8
+  container_config_lambda_node12: &container_config_lambda_node12
     working_directory: ~/project/build
     docker:
       - image: lambci/lambda:build-nodejs8.10
```
Comment (joelcarr) [inline .circleci/config.yml]: **Chore:** We should update the version number here too.

**Review by AniaMakes — APPROVED** [review]
🧹

File: .circleci/config.yml (hunk 1 above)
Comment (magsallen) [reply .circleci/config.yml]: Cheers!

**Review by joelcarr — APPROVED** [review]
🧹

**Pull request merged**